Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -35,16 +35,11 @@ your web root. At the moment, only these few lines are required:
                 "type": "composer",
                 "url": "https://composer.typo3.org/"
             }
-        ],
-        "extra": {
-            "typo3/cms": {
-                "cms-package-dir": "{$vendor-dir}/typo3/cms",
-                "web-dir": "public"
-            }
-        }
+        ]
     }
 
-You must set the correct name of your web root folder in property ``web-dir``.
+Note, that you only need to include this composer repository, if you have to use
```

Comment:
I'd love to see properly cased "Composer" everywhere. ;-)

Reply:
done